tests/unit/test_charm_scaling.py

```diff
@@ -165,6 +163,33 @@ def test_scaling_instance_map_configured(harness: Harness) -> None:
                 "port": 8034,
             },
         }
+    worker_config_path = root / synapse.SYNAPSE_WORKER_CONFIG_PATH[1:]
+    with open(worker_config_path, encoding="utf-8") as config_file:
+        content = yaml.safe_load(config_file)
+        assert content["worker_name"] == "federationsender1"
```

Comment:
Perhaps use the constant that you created for it

Reply:
No my bad, the constant is for the service name, but might be good to use a constant for "federationsender1" it's repeated at 3 or 4 places

Reply:
Good point, suggest that I do this in a separate PR.